tests/TestPC.py

```diff
@@ -4,12 +4,15 @@
 import unittest
 import hashlib
 import numpy as np
+np.random.seed(42)
```

Comment:
how about we have two tests here?

Also please do not fix random seed for whole file here.

How about we have one test with seed fixed, one test without seed fixed? And only fix seed in the fix-seed function?

Reply:
√